tests/Support/SupportNumberTest.php

```diff
@@ -123,6 +123,18 @@ public function testToCurrency()
         $this->assertSame('$5.32', Number::currency(5.325));
     }
 
+    public function testToCurrencyWithPrecision()
+    {
+        $this->needsIntlExtension();
+
+        $this->assertSame('$0', Number::currency(0, precision: 0));
+        $this->assertSame('$2', Number::currency(1.50, precision: 0));
+
+        $this->assertSame('-$2', Number::currency(-1.75, precision: 0));
```

Comment:
@dwightwatson Do we have a way to control the rounding mode her? As i don't think that silently rounding up to the nearest integer is what is actually wanted when dealing with money precision wise?

Reply:
Yeah, so it looks like it does support a `ROUNDING_MODE` and then there are a heap of constants to control the direction. I'm not sure the nicest way to expose this in the helper without having to take in one of the constants. Do you have any ideas?

Reply:
Also worth considering that if there's reason enough to support controlling rounding it should probably also be added to `Number::format()` for consistency.

Reply:
Definitely like this PR, but I should mention that I think as soon as we start rounding currencies, we are entering dangerous territory. To be fair, just using floating point numbers for currency is also living on the edge in my opinion.

Reply:
While looking for inspiration on how to handle rounding I did note that [`number_to_currency` in Rails](https://apidock.com/rails/v6.1.3.1/ActionView/Helpers/NumberHelper/number_to_currency) also supports precision without control over the rounding direction.